### DISCUSSION TITLE
Working Vehicles

### DISCUSSION THREAD

**Discussion by Blueion76** [Blueion76/FCAUconnect-HA#2] [Show and tell]
If you have a vehicle that's working successfully with this addon, post the year, make and model here to let others know :)

Your settings could also be helpful, such as letting others know what country you're in and what region you chose in the addon settings.

**Comment by withay** [comment]
2024 Alfa Romeo Tonale Veloce - works great!

**Reply by redwolftech** [reply]
Where are you located? (not working for me with a Giulia in the US - just get invalid LoginID although following the link given and doing so manually works fine)

**Reply by withay** [reply]
I'm in New Hampshire - still ok for me - I actually used it to start the car this afternoon.

**Reply by redwolftech** [reply]
I poked around some more with mine - I actually think it may be because you have a Tonale - seems like they have changed how the car works on this connected stuff in the Tonale but haven't updated the Giulia (I have something similar, but actually seems to be a different "Alfa Connect" and not the same uconnect powered stuff?).

**Comment by Blueion76** [comment]
2024 Jeep Wagoneer L - forgot to add my own vehicle haha.

**Comment by Flashlab2** [comment]
2022 Fiat 500e Cabrio, Europe works!

**Comment by The-ScubaScott** [comment]
Working on 2023 Ram 1500 and got some bad ass automations working now too! Thanks!!!!

**Comment by kenirod** [comment]
2024 Abarth 500e, europe work great!  Thank you very much!

**Comment by kamushadenes** [comment]
2022 Jeep Compass S T270, Brazil - works flawlessly!

**Comment by The-ScubaScott** [comment]
I use it constantly but I have a RAM 1500
